resources/assets/lib/store-checkout.coffee

```diff
@@ -37,13 +37,14 @@ export class StoreCheckout
 
     $(@CHECKOUT_SELECTOR).on 'click.checkout', (event) =>
       provider = event.target.dataset.provider
+      orderId = event.target.dataset.orderId
```

Comment:
looks like some already have `orderNumber`, maybe use that instead?

Reply:
`orderNumber` is for external reference